<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>599</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fbc升级工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Fbc升级工具</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionUpgradeStting"/>
   <addaction name="actionUpgrade"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/question.png</normaloff>:/images/question.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
   <property name="toolTip">
    <string>连接串口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>断开串口</string>
   </property>
   <property name="toolTip">
    <string>断开串口连接</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
   <property name="toolTip">
    <string>配置串口</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/cleanport.png</normaloff>:/images/cleanport.png</iconset>
   </property>
   <property name="text">
    <string>清空数据</string>
   </property>
   <property name="toolTip">
    <string>清空控制台数据</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionUpgrade">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/upgrade.png</normaloff>:/images/upgrade.png</iconset>
   </property>
   <property name="text">
    <string>开始升级</string>
   </property>
   <property name="toolTip">
    <string>点我升级</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionUpgradeStting">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/upgrade_setting.png</normaloff>:/images/upgrade_setting.png</iconset>
   </property>
   <property name="text">
    <string>升级设置</string>
   </property>
   <property name="toolTip">
    <string>升级配置</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>点我保存Console中日志文件</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="icon">
    <iconset resource="terminal.qrc">
     <normaloff>:/images/unlock.png</normaloff>:/images/unlock.png</iconset>
   </property>
   <property name="text">
    <string>LockStatus</string>
   </property>
   <property name="toolTip">
    <string>空片或者Spi写坏请点我</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
